### ISSUE TITLE
Admin URL

### ISSUE THREAD

**Issue by wratke** [area17/twill#29]
I'm trying to setup the admin routing, but I'm running into some issues. In the config/twill.php file I've added this to override the default setup:

return [

    /*
    |--------------------------------------------------------------------------
    | Application Admin URL
    |--------------------------------------------------------------------------
    |
    | This value is the URL of your admin application.
    |
     */
    'admin_app_url' => env('ADMIN_APP_URL', env('APP_URL').'/admin')
];

The app URL is set to http://nin-twill.loc/ in the .env file. When I try to load the admin section by going to http://nin-twill.loc/admin, I get a NotFoundHttpException error. Am I missing something?

**Comment by nikhiltri** [comment]
Is there an extra slash getting added between the APP_URL and /admin? I see that the APP_URL ends in a slash.

**Comment by luislavena** [comment]
If not mistaken, you should be using `ADMIN_APP_PATH` instead (https://github.com/area17/twill/issues/5#issuecomment-403534753)

**Comment by wratke** [comment]
I've tried following the advice from that issue, but I'm stilling having problems. The ADMIN_APP_URL and ADMIN_APP_PATH were missing from the .env file, so I added them both in, set ADMIN_APP_URL to be the same as the APP_URL (both nin-twill.loc), and set ADMIN_APP_PATH to admin. I removed what I'd added to the config/twill.php file, it's returning an empty array. When I go to nin.twill.loc/admin I get the same error I was getting before. Do I need to add anything to the admin.php routes file? Right now I just have a module that I added called projects: Route::module('projects');
